app/grants/views.py

```diff
@@ -2247,3 +2252,18 @@ def add_grant_from_collection(request, collection_id):
     return JsonResponse({
         'grants': grants,
     })
+
+
+@login_required
+@staff_member_required
```

Comment:
^ why does need a login + staff decorator?

Reply:
This Is for development and debug purposes like the email views.